### DISCUSSION TITLE
serverless-slic-watch-plugin depends on serverless version conflicts with disable Logs: true

### DISCUSSION THREAD

**Discussion by 1611176299** [serverless/serverless#10022] [Q&A]
When I use serverless@<2.10.0 and disable Logs: true, my serverless works normally, but I need severe@>=2.30.0 when I use serverless-slic-watch-plugin, but it will appear when my severeless creates a stack:

> An error occurred: IamRoleLambdaExecution - Policy statement must contain resources. (Service: AmazonIdentityManagement; Status Code: 400; Error Code: MalformedPolicyDocument; Request ID: 0d57ac37-5ce8-4bd7-9f8e-33cd43b7fdcd; Proxy: null).

> "PolicyDocument":{
> "Version": "2012-10-17",
> "Statement": [
> {
> "Effect": "Allow",
> "Action": [
> "logs:CreateLogStream",
> "logs:CreateLogGroup"
> ],
> "Resource": []
> },
> {
> "Effect": "Allow",
> "Action": [
> "logs:PutLogEvents"
> ],
> "Resource": []
> },

**Comment by pgrzesik** [comment]
Hello, could you provide more details about the issue? Ideally with a small reproducible example. Did you report this problem on the plugin side?
